src/sdl/point.hpp

```diff
@@ -20,35 +20,15 @@
 #include <iosfwd>
 #include <tuple>
 
-/** Holds a 2D point. */
-struct point
+/** Holds a 2D point. This is a thin wrapper over SDL_Point. */
+struct point : SDL_Point
```

Comment:
Doesn't this need a `public`? Or is that the default because it's a struct…?

Reply:
That's right, it doesn't need public because it's the default for a struct.